### ISSUE TITLE
10.13 NBI: Auto Login Sometimes Fails

### ISSUE THREAD

**Issue by macmule** [macmule/AutoCasperNBI#108]
When booting to 10.13, auto-login sometimes fails

**Comment by macmule** [comment]
Looks to be an Apple issue...

TL;DR use NFS unless running Server.app on High Sierra, in which case [read this](http://www.openradar.me/34639788)

## Open Radar
As per: [http://www.openradar.me/34640065](http://www.openradar.me/34640065)

## Summary:
Tested over Server.app: 5.2, 5.3 & 5.4.

When a 10.13.x NetBoot Image, with auto-login enabled, is served over HTTP. Auto-Login fails.

The same 10.13.x NetBoot Image, will auto-login when NFS is enabled (see 34639788 for the related issue with this)

## Steps to Reproduce:
1. Create an NBI with auto-login enabled for a user account
2. Add to NetInstall Service
3. Enable NetInstall Service
4. Share NBI over HTTP
5. Attempt to Boot from NBI
6. Cancel NetBoot
7. Share NBI over NFS
8. Attempt to Boot from NBI

## Expected Results:
Auto-login should work on the NBI regardless of protocol shared over.

## Actual Results:
HTTP fails to auto-login, NFS works

## Version:
10.13.x

**Comment by danprk** [comment]
I built a 10.13.1 nbi and it's not auto logging in, jamf just sits at the temporary adobe install account lock screen.

**Comment by macmule** [comment]
@danprk ok.. but the NBI logged in? The  AdobeInstall account is a jamf issue & not one for this project.

**Comment by danprk** [comment]
@macmule nbi logged in fine with nfs, sorry i misread this issue

**Comment by macmule** [comment]
Looks to have been resolved within a more recent 10.13.x update
